src/utils/hooks/useClickOutside.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+
+/**
+ * useClickOutside - Handle click outside a referenced element (e.g. Popover)
+ */
+type UseClickOutside = (
+    ref: { current: HTMLElement | null | undefined },
+    handler: (ev: React.KeyboardEvent<HTMLDivElement>) => void
+) => void;
+
+const useClickOutside: UseClickOutside = (ref, handler) => {
+    useEffect(() => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
what are those errors? Can we resolve them?

Reply:
this comment has been removed in latest commits